wiki/serve_wiki.py

```
@@ -0,0 +1,94 @@
+    }
+
+    def guess_type(self, path):
+        ext = Path(path).suffix.lower()
+        return self.MIME_TYPES.get(ext, 'application/octet-stream')
+
+    def end_headers(self):
+        self.send_header('Cache-Control', 'no-cache, no-store, must-revalidate')
+        self.send_header('Access-Control-Allow-Origin', '*')
+        super().end_headers()
+
+    def log_message(self, format, *args):
```

Ruff (0.15.13)

[error] Function argument `format` is shadowing a Python builtin

(A002)